### DISCUSSION TITLE
Pinterest长按无法呼出扇形菜单

### DISCUSSION THREAD

**Discussion by menxico** [gkd-kit/gkd#393] [Q&A]
给GKD打开无障碍权限后，Pinterest的长按呼出扇形菜单消失了，关闭无障碍权限后又可以正常呼出扇形菜单正常pin图了，这个问题怎么解决？![Screenshot_2023-12-16-22-50-51-108_com.pinterest.jpg](https://github.com/gkd-kit/gkd/assets/51862631/d9d16000-374f-4f90-95ea-f53289f5f747)

**Comment by lisonge** [comment]
关闭无障碍前台

**Reply by menxico** [reply]
关闭了也没有解决，只能停用了。
